fog/sql_recovery_db/src/lib.rs

```diff
@@ -1220,8 +1217,8 @@ impl SqlRecoveryDb {
                     return Ok(result);
                 }
 
-                let mut report_bytes = Vec::with_capacity(data.report.encoded_len());
-                data.report.encode(&mut report_bytes)?;
+                let mut report_bytes = Vec::with_capacity(data.attestation_evidence.encoded_len());
+                data.attestation_evidence.encode(&mut report_bytes);
```

Comment:
`AttestationEvidence` is a `OneOf` and in prost the `OneOf::encode()` is infallible, even though the `Message::encode()` is fallible :shrug:

Reply:
NVM this got reworked to no longer use OneOf directly since OneOf doesn't support direct decoding